lumicks/pylake/kymotracker/detail/peakfinding.py

```diff
@@ -24,13 +29,17 @@ def find_kymograph_peaks(
     rect : tuple, optional
         Rectangle to crop results to. Tuple of integer pixel values referring to a pair of
         (time, position) coordinates. Default: no cropping.
+    filter_width : float
+        Width of the point spread function in pixels. Default: 0.5.
     """
-    coordinates, time_points = peak_estimate(kymograph_data, half_width_pixels, threshold)
+    data = kymograph_data.astype(float, copy=False)
```

Comment:
By default, the filter `scipy.ndimage.gaussian_filter()` keeps the output format the same as the input format, hence you lose precision right after you blur. Considering that this function is intended to be used by more than one tracking algorithm in the future, I figured it'd be a good idea to be a bit defensive about the input datatype. However, I see that since scipy `1.9`, the `output` parameter also takes an optional `dtype`, so that is probably a better option here.